packages/advi/advi.2.0.0/opam

```diff
@@ -22,25 +21,24 @@ authors: [
 ]
 license: "LGPL-2.1-or-later"
 homepage: "http://advi.inria.fr"
-dev-repo: "git+https://github.com:diremy/advi.git"
+dev-repo: "git+https://github.com/diremy/advi.git"
```

Comment:
The previous syntax used does not work. The semicolon is usually used in conjunction with the ssh syntax (e.g. `git@github.com:diremy/advi.git`) which is only accessible to people with write access